<launch>
 


    <node pkg="save_image" type="CameraStream" name="CameraStream" output="screen" >
	<param name="device" type="int" value="0" />
	<param name="width" type="double" value="480" />
	<param name="height" type="double" value="640" />
	<param name="fps" type="double" value="30" />
	<param name="real_time" type="bool" value="true"/>
	<param name="img_path" type="string" value="/home/ste/Immagini/test_drone/"/>
	<param name="img_path_save" type="string" value="/home/ste/"/>
	<param name="save_img" type="bool" value="false"/>
    </node>


        
</launch>
